packages/superdough/superdough.mjs

```diff
@@ -288,24 +284,25 @@ export const superdough = async (value, t, hapDuration) => {
     postgain = 1,
     density = 0.03,
     // filters
-    ftype = '12db',
+
     fanchor = 0.5,
+    drive = 0.69,
     // low pass
     cutoff,
     lpenv,
     lpattack,
     lpdecay,
     lpsustain,
     lprelease,
-    resonance = 1,
+    resonance = 0.5,
```

Comment:
is this still needed after taming the resonance curve? would be a breaking change

Reply:
updated the default back to normal :)